docs/GraFx-Studio/concepts/connectors-media/index.md

```diff
@@ -0,0 +1,32 @@
+# Media Connectors
+
+**Media Connectors** in GraFx Studio enable the integration of external media providers. These connectors offer access to a range of systems, including DAM (Digital Asset Management) platforms, as well as dynamic asset providers such as generative AI for images, real-time map generators, and dynamic chart generators. By leveraging Media Connectors, users can effectively utilize a diverse array of media assets directly within GraFx Studio.
+
+![ui](connector2.png)
+
+## Support
+
+CHILI publish categorizes CHILI GraFx Connectors into four categories.
+
+### Built-in
+
+GraFx Studio connects to GraFx Media through the GraFx Media connector, utilizing the same framework as other media connectors. CHILI publish assumes full responsibility for the development and support of the GraFx Media connector and the associated media service, as all elements were built in-house.
```

Comment:
@BramVerniest maybe link to https://docs.chiligrafx.com/GraFx-Media/?

Reply:
done